static-assets/components/cstudio-admin/mods/admin-configurations.css

```diff
@@ -15,73 +15,128 @@
  */
 
 .configuration-window {
-    position: relative;
-    top: 15px;
-    padding-left:5%;
-    right:auto;
-    width:100%;
+  position: relative;
+  top: 15px;
+  padding-left: 5%;
+  right: auto;
+  width: 100%;
 }
 
 #config-list {
-	margin: 10px 0 10px 0;
+  margin: 10px 0 10px 0;
 }
 
 #menu-area {
-	display: inline-block;
-	margin: 10px 0 10px 0;
-    width: 95%;
+  display: inline-block;
+  margin: 10px 0 10px 0;
+  width: 95%;
 }
 
 #config-area {
-	padding-top: 55px;
+  padding-top: 55px;
 }
 
 #config-description {
-	float: left;
+  float: left;
 }
 
 #config-buttons {
-	float: right;
+  float: right;
 }
 
 .edit-button {
-    width: 100px;
-    background: #000000;
-    color: #FFFFFF;
-    border: none;
-    margin: 5px;
-    height: 20px;
+  width: 100px;
+  background: #000000;
+  color: #FFFFFF;
+  border: none;
+  margin: 5px;
+  height: 20px;
 }
 
 #edit-window {
-	margin: 5px 0 5px 0;
-	float: left;
-    width: 95%;
+  margin: 5px 0 5px 0;
+  float: left;
+  width: 95%;
 }
 
-.sample #edit-window{
-    margin: 5px 5% 5px 0;
-    width: 45%;
+.sample #edit-window {
+  margin: 5px 5% 5px 0;
+  width: 45%;
 }
 
 #edit-window .editor-text {
-	height: 600px;
-	resize: none;
-	width: 100%;
-    font-family: monospace, sans-serif;
+  height: 600px;
+  resize: none;
+  width: 100%;
+  font-family: monospace, sans-serif;
 }
 
 #sample-window {
-    float: left;
-    height: 100%;
-    width: 45%;
-    margin: 5px 0 5px 0;
+  float: left;
+  height: 100%;
+  width: 45%;
+  margin: 5px 0 5px 0;
 }
 
 #sample-text {
-	height: 600px;
+  height: 600px;
 }
 
-#edit-window .CodeMirror pre, #sample-window .CodeMirror pre{
-    overflow: initial;
-}
\ No newline at end of file
+#edit-window .CodeMirror pre, #sample-window .CodeMirror pre {
+  overflow: initial;
+}
+
+.hint-btn {
+  padding: 6px;
+  vertical-align: middle;
+}
+
+.hint-btn .hint-btn--icon, #encryptHintText .hint-text--icon {
+  font-size: 20px;
+  color: #5cc7fa;
+}
+
+#encryptHintText {
+  width: 95%;
+  display: flex;
+  padding-bottom: 60px;
+}
+
+#encryptHintText .hint-text--icon {
+  margin-right: 10px;
+  font-size: 14px;
+  margin-top: 4px;
+}
+
+#encryptHintText .hint-text--icon::before {
+  border: 2px solid;
+  padding: 2px 6px;
+  border-radius: 10px;
+}
+
+#encryptHintText .hint .title {
+  font-size: 12px;
+  margin: 2px 0 0;
+}
+
+#encryptionInfoDialog .encrypt-hint {
+  display: flex;
+}
+
+#encryptionInfoDialog .encrypt-hint .encrypt-hint--icon {
+  font-size: 12px;
+  color: #5cc7fa;
+  margin-right: 10px;
+  margin-top: 2px;
+}
+
+#encryptionInfoDialog .encrypt-hint .encrypt-hint--icon::before {
+  border: 2px solid;
+  padding: 2px 6px;
+  border-radius: 10px;
+}
+
+#encryptionInfoDialog .encrypt-hint .encrypt-hint--description {
+  padding: 0;
+  margin: 0;
+}
```

Comment:
The idea is that when you have specific classes, you don't need to nest styles.

`#encryptionInfoDialog .encrypt-hint .encrypt-hint--description` => `.encryption-info-dialog--encrypt-hint--description`